rust/rbac-registration/src/cardano/cip509/utils/cip134/mod.rs

```diff
@@ -0,0 +1,7 @@
+//! Utilities for [CIP-134] (Cardano URIs - Address Representation).
```

Comment:
Move the `Cip0134Uri` type to the `cardano-blockchain-types` crate.  Its not exclusively part of RBAC, its just used by RBAC.